src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -1367,4 +1367,39 @@ public void testDataSourceOktaGenerates429StatusCode() throws Exception {
       thread.join();
     }
   }
+
+  @Test
+  @ConditionalIgnoreRule.ConditionalIgnore(condition = RunningOnGithubAction.class)
```

Comment:
why do we need to mark these tests to not run on GH?

Reply:
That was a mistake. I fixed this in the latest commit.